base/abstractarray.jl

```diff
@@ -461,12 +461,16 @@ end
 """
     last(coll)
 
-Get the last element of an ordered collection, if it can be computed in O(1) time. This is
-accomplished by calling [`lastindex`](@ref) to get the last index. Return the end
-point of an [`AbstractRange`](@ref) even if it is empty.
+Get the last element of an ordered collection, if it can be computed by reversing the collection. This is
```

Comment:
I'd probably leave this line as-is, since the O(1) time is the most relevant part here and reversing the collection could theoretically also mean collecting the whole iterator.
```suggestion
Get the last element of an ordered collection, if it can be computed in O(1) time. This is
```